azure/pipelines/node-npm/install-volta.yml

```diff
@@ -0,0 +1,35 @@
+# Install Volta Node version manager in your pipeline
+# https://wiki.delaware.pro/pages/viewpage.action?pageId=314778333
+
+steps:
+- bash: |
+   echo "Downloading Volta..."
+   curl https://get.volta.sh | bash -s -- --skip-setup
+
+   echo "Preparing environment variables..."
+   export VOLTA_HOME="$HOME/.volta"
+   export PATH="$VOLTA_HOME/bin:$PATH"
+   
+   echo "##vso[task.setvariable variable=VOLTA_HOME]$VOLTA_HOME"
+   echo "##vso[task.setvariable variable=PATH]$PATH"
+   
+   echo "Starting installation..."
+   volta setup --verbose
+  condition: or(eq( variables['Agent.OS'], 'Linux' ), eq( variables['Agent.OS'], 'Darwin' ))
```

Comment:
Can't we use compile-time expression like e.g. [dotnet-ci](https://github.com/delawarePro/platina-scripts/blob/master/azure/pipelines/dotnet/ci.yml#L90]), so that we don't see this step when not applicable?

Reply:
I tried this after your comment but can't get it to work. I found this on SO
https://stackoverflow.com/questions/71664997/condition-for-template-using-agent-os-agent-os

Does this seem valid?

I tried this in the template
`parameters:
  - name: voltaVersion
    displayName: Volta Version
    type: string
    default: '1.1.0'

steps:
  - ${{ if or(eq( variables['Agent.OS'], 'Linux' ), eq( variables['Agent.OS'], 'Darwin' )) }}:
      - bash: |
          echo "Downloading Volta..."
          curl https://get.volta.sh | bash -s -- --version ${{parameters.voltaVersion}} --skip-setup

          echo "Preparing environment variables..."
          export VOLTA_HOME="$HOME/.volta"
          export PATH="$VOLTA_HOME/bin:$PATH"

          echo "##vso[task.setvariable variable=VOLTA_HOME]$VOLTA_HOME"
          echo "##vso[task.setvariable variable=PATH]$PATH"

          echo "Starting installation..."
          volta setup --verbose

        displayName: Install Volta for Unix

  - ${{ if eq( variables['Agent.OS'], 'Windows_NT' ) }}:
      - powershell: |
          $voltaFileName = "volta-${{parameters.voltaVersion}}-windows-x86_64.msi"
          Write-Host "Downloading ${voltaFileName}..."
          Invoke-WebRequest https://github.com/volta-cli/volta/releases/download/v${{parameters.voltaVersion}}/${voltaFileName} -OutFile $(System.DefaultWorkingDirectory)\${voltaFileName}

          Write-Host "Starting installation..."
          Start-Process $(System.DefaultWorkingDirectory)\${voltaFileName} -ArgumentList "/quiet" -Wait

          Write-Host "Updating PATH..."
          $env:PATH = "C:\Program Files\Volta\;$env:PATH"
          Write-Host "##vso[task.setvariable variable=PATH]C:\Program Files\Volta\;$env:PATH"

          Write-Host $env:PATH

        displayName: Install Volta for Windows`